takwimu/takwimu_ui/public/static/img/flags/beta.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="37" viewBox="0 0 38 37">
```

Comment:
QQ: Are there any benefit of putting these flags in public folder directly? We'll be losing a lot: https://facebook.github.io/create-react-app/docs/using-the-public-folder#adding-assets-outside-of-the-module-system

Reply:
These are for testing, i think I've figured a better way of doing this.